manifests/uno.ui-preview-major.manifest.json

```diff
@@ -3,12 +3,12 @@
     "toolVersion": "1.14.0",
     "variables": {
       "OPENJDK_VERSION": "11.0.20.1",
-      "DOTNET_SDK_VERSION": "9.0.100-preview.6.24328.19",
-      "MACCATALYST_SDK_VERSION": "17.2.9714-net9-p6/9.0.100-preview.6",
-      "IOS_SDK_VERSION": "17.2.9714-net9-p6/9.0.100-preview.6",
-      "MACOS_SDK_VERSION": "14.2.9714-net9-p6/9.0.100-preview.6",
-      "ANDROID_SDK_VERSION": "34.99.0-preview.6.340/9.0.100-preview.6",
-      "MAUI_VERSION": "9.0.0-preview.6.24327.7/9.0.100-preview.6"
+      "DOTNET_SDK_VERSION": "9.0.100-preview.7.24407.12",
+      "MACCATALYST_SDK_VERSION": "17.5.9231-net9-p7/9.0.100-preview.7",
+      "IOS_SDK_VERSION": "17.5.9231-net9-p7/9.0.100-preview.7",
+      "MACOS_SDK_VERSION": "14.5.9231-net9-p7/9.0.100-preview.7",
+      "ANDROID_SDK_VERSION": "35.0.0-preview.7.41/9.0.100-preview.7",
```

Comment:
https://www.nuget.org/packages/Microsoft.NET.Sdk.Android.Manifest-9.0.100-preview.7 is now bumped to 35.x.

@ebariche Does it mean anything for us? There are some 34.x reference in the manifest, I assume these may need to be updated?